libs/ui/src/lib/drag-and-drop-file-input/drag-and-drop-file-input.component.html

```diff
@@ -0,0 +1,5 @@
+<ngx-dropzone multiple="false" (change)="selectFile($event)">
+  <ngx-dropzone-label *ngIf="!fileName">{{ placeholder }}</ngx-dropzone-label>
```

Comment:
The placeholder should be internationalized. check out CONTRIBUTING doc i18n for more info, it should be a key, then translated in the template.